docs/src/sway-on-chain/libraries.md

```diff
@@ -34,6 +36,48 @@ pub struct MyStruct {
 }
 ```
 
+Libraries are composed of just a Forc.toml and src folder, unlike usual Sway projects which usually contain tests and a Cargo.toml. An example of a Library's Forc.toml 
+
+```toml=
+[project]
+author = "Fuel Labs <contact@fuel.sh>"
+entry = "lib.sw"
+license = "Apache-2.0"
+name = "lib-std"
+
+[dependencies]
+"core" = { git = "http://github.com/FuelLabs/sway-lib-core" }
+```
+
+which denotes the author, the name by which it can be imported, an entry file, and any dependencies. For large libraries, it is recommended to have a lib.sw entry point that depends on all of the library's functionality. For example, the lib.sw of the standard library looks like
```

Comment:
```suggestion
which denotes the author, an entry file, the name by which it can be imported, and any dependencies. For large libraries, it is recommended to have a `lib.sw` entry point that depends on all of the library's functionality. For example, the `lib.sw` of the standard library looks like:
```